container/BUILD

```diff
@@ -130,18 +130,11 @@ TEST_TARGETS = [
     ":cc_based_warning",
     "/pause_piecemeal:image",
     "/pause_piecemeal_gz:image",
-    ":cc_image",
```

Comment:
Is it possible to fully remove the cc_image target and some of the below targets now from https://github.com/bazelbuild/rules_docker/blob/master/testdata/BUILD#L756?